tests/gaudi/l2/hccl_build.yaml

```diff
@@ -13,29 +13,20 @@ spec:
   output:
     to:
       kind: ImageStreamTag
-      name: 'hccl-demo-workload:latest'
+      name: 'hccl-demo-workload:1.18.0-524'
   strategy:
     type: Docker
   source:
     type: Dockerfile
     dockerfile: |
-      ARG BUILDER=vault.habana.ai/gaudi-docker/1.17.1/rhel9.4/habanalabs/pytorch-installer-2.3.1:1.17.1-40
+      ARG BUILDER=vault.habana.ai/gaudi-docker/1.18.0/rhel9.4/habanalabs/pytorch-installer-2.4.0:1.18.0-524
       FROM ${BUILDER} AS builder
 
       WORKDIR /
       RUN git clone https://github.com/HabanaAI/hccl_demo.git \
           && cd hccl_demo \
           && make
 
```

Comment:
why do we remove the hccl_ofi_wrapper? it is not needed anymore in 1.18?

Reply:
@uMartinXu  It is not needed in 1.18 as it is already included in base image at `/usr/lib/habanalabs/libhccl_ofi_wrapper.so`